src/main/kotlin/no/nav/arbeidsgiver/min_side/userinfo/UserInfoController.kt

```diff
@@ -132,20 +135,31 @@ class UserInfoController(
                 }
             }.await()
 
-            tilganger to organisasjoner
+            val syfoVirksomheter = async {
+                runCatching {
+                    digisyfoService.hentVirksomheterOgSykmeldte(authenticatedUserHolder.fnr)
+                }
+            }.await()
```

Comment:
Nå har vi 
```kotlin
val tilganger = A.awaitAll()
val organisasjoner = B.await()
val syfoVirksomheter = C.await()
return Triple(tilganger, organisasjoner, syfoVirksomheter)
```
så de tre kjører sekvensielt. For å få dem parallelt kan man skrive om til:
```kotlin
val tilganger = A
val organisasjoner = B
val syfoVirksomheter = C
return Triple(tilganger.awaitAll(), organisasjoner.await(), syfoVirksomheter.await())
```

Reply:
nice! det burde jeg ha tenkt på :)

Reply:
dea70ce5f1f7ad7ef1e5cdbf484f46b0d763d9f6